roles/tackle/tasks/main.yml

```diff
@@ -95,12 +95,10 @@
         - name: "Generate random values for Keycloak SSO credentials"
           set_fact:
             keycloak_sso_admin_password: "{{ lookup('password', '/dev/null chars=ascii_lowercase,ascii_uppercase,digits length=16') }}"
-            keycloak_sso_hub_client_secret: "{{ lookup('password', '/dev/null chars=ascii_lowercase,ascii_uppercase,digits length=32') }}"
 
         - name: "Encode Keycloak SSO credentials"
           set_fact:
             keycloak_sso_admin_password_b64: "{{ keycloak_sso_admin_password | b64encode }}"
-            keycloak_sso_hub_client_secret_b64: "{{ keycloak_sso_hub_client_secret | b64encode }}"
```

Comment:
We also need to remove the Realm configmap creation below since it is getting deleted